src/services/codefixes/importFixes.ts

```diff
@@ -101,7 +103,9 @@ namespace ts.codefix {
                     doAddExistingFix(changes, sourceFile, importClause, defaultImport, namedImports, canUseTypeOnlyImport);
                 });
                 newImports.forEach((imports, moduleSpecifier) => {
-                    addNewImports(changes, sourceFile, moduleSpecifier, quotePreference, imports);
+                    // Add blank line after last new import statement
+                    const blankLineBetween = lastModuleSpecifier === moduleSpecifier;
+                    addNewImports(changes, sourceFile, moduleSpecifier, quotePreference, imports, blankLineBetween);

```

Comment:
I think this is clear enough, and shorter.

```suggestion
                    addNewImports(changes, sourceFile, moduleSpecifier, quotePreference, imports, /*blankLineBetween*/ lastModuleSpecifier === moduleSpecifier);
```

Reply:
@sandersn Thanks for the feedback. Done.